project-rome-docs/objectivec-api/userdata/MCDUserDataFeed.md

```diff
@@ -24,6 +24,11 @@ Describes the current status of user data synchronization.
 
 Event for when the sync status of the UserDataFeed changes.
 
+### daysToSync
+`@property(nonatomic, readwrite) NSInteger daysToSync;`
+
+The number of days of data to sync, which should be fewer than 30 represents the default value, which will be determined by the server.
```

Comment:
This looks like it should be two sentences?

Reply:
Yep!